src/App.js

```diff
@@ -70,11 +70,36 @@ function App() {
         if (user) {
           setAuthenticatedUser(user);
 
-          // localStorage.setItem("user", JSON.stringify(user));
+          localStorage.setItem("user", JSON.stringify(user));
         }
       });
   };
 
+  function getProfile() {
+    const token = localStorage.getItem("Token");
+    console.log("Token", token);
+
+    const fetchOptions = {
+      method: "GET",
+      headers: {
+        authorization: token,
+      },
+    };
+
+    fetch(`http://localhost:3030/users/profile`, fetchOptions)
+      .then((res) => res.json())
+      .then((data) => {
+        setProfile(data.profile);
+      })
+      .catch((error) => {
+        console.error(error);
+      });
+  }
+
+  useEffect(() => {
+    getProfile();
+  }, [authenticatedUser]);
```

Comment:
It doesn't look like you need `authenticatedUser` as a dependency here. It can just be an empty array.

Reply:
ok!